ServiceBoxWeb/src/main/resources/templates/employee/service-order-list.html

```diff
@@ -0,0 +1,43 @@
+<!doctype html>
+<html lang="pl" xmlns:th="http://www.thymeleaf.org">
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>ServiceBox</title>
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.0/dist/css/bootstrap.min.css" rel="stylesheet"
+          integrity="sha384-gH2yIJqKdNHPEq0n4Mqa/HGKIhSkIHeL5AyhkYV8i59U5AR6csBvApHHNl/vI1Bx" crossorigin="anonymous">
+    <link th:href="@{/css/style.css}" rel="stylesheet" type="text/css">
+</head>
+<body>
+<div class="container">
+    <div class="row">
+        <div class="col-md-12">
+            </br>
+            </br>
+            <table class="table table-bordered table-hover table-responsive">
+                <tr>
+                    <th th:text="#{so.orderNumber}"></th>
+                    <th th:text="#{so.createdAt}"></th>
+                    <th th:text="#{so.maxCost}"></th>
+                    <th th:text="#{so.description}"></th>
+                    <th th:text="#{so.onlyNewParts}"></th>
+                    <th th:text="#{so.state}"></th>
+                </tr>
+                    <tr th:each="service : ${service}">
+                    <td th:text="${service.orderNumber}"></td>
+                    <td th:text="${service.createdAt}"></td>
+                    <td th:text="${service.maxCost}"></td>
+                    <td th:text="${service.description}"></td>
+                    <td th:if="${service.onlyNewParts}" th:text="#{form.yes}"></td>
+                    <td th:unless="${service.onlyNewParts}" th:text="#{form.no}"></td>
+                    <td th:text="${service.state}"></td>
```

Comment:
Enum posiada opisy statusów - można wykorzystać getter > getDescription
`<td th:text="${service.state.description}"></td>`

Reply:
Poprawiono, ależ nie sądziłam że z tym description będzie to takie banalne ;-).